src/WebJobs.Script.WebHost/ContainerManagement/LinuxContainerInitializationHostService.cs

```diff
@@ -15,7 +11,7 @@
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.ContainerManagement
 {
-    public class LinuxContainerInitializationHostService : IHostedService
+    public abstract class LinuxContainerInitializationHostService : IHostedService
```

Comment:
rename class to `LinuxContainerInitializationHostedService`  as this is an implementation of `IHostedService` see similar naming convention here - `https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/NullHostedService.cs`